modules/tools/pages/udfs-ui.adoc

```diff
@@ -0,0 +1,70 @@
+= User-Defined Functions UI
+:description: The Couchbase Server admin console provides a UI for adding User-defined-functions
+
+[abstract]
+{description}
+
+
+Starting from Version 7.1, the Couchbase Administration console provides a CRUD{empty}footnote:[**CR**eate **U**pdate **D**elete] interface for user defined functions written in {sqlpp} or JavaScript.
+
+Start at the administration console and select menu:Query[UDFs] from the menus to access the `UDF` screen:
+
+image::udfs-ui/udfs-screen.png[alt="screen for adding user-defined functions"]
+
+== Javascript Function Libraries
+
+Click on `{plus}{nbsp}add function library` link to create a library of JavaScript functions. 
+
+image::udfs-ui/add-library-screen.png[,500, alt="add javascript library screen"]
+
+When you create a new library, an example function (`add`) is supplied as an example. 
+You can delete it when you add your own function, or leave it to serve as a basic example.
+
+You can set a `Namespace` for your library which will restrict its access to users that have permissions to access the bucket and scope defined in the Namespace.
+
+Before you leave the screen, you must set a `Library Name`.
+
+Press the btn:[Save] to leave the editor and return to the main UDF screen.
+
+image::udfs-ui/udf-screen-with-library-added.png[]
+
+You can `edit` the library again to add and/or delete functions contained inside it. 
+
+You can also delete the whole library by clicking on the `Drop` link.
+
+[#user-defined-functions]
+== User-Defined Functions
+
+You can add your own user-defined functions in the lower `User-Defined Functions` list. These functions can be one of two types:
+
+[horizontal]
+*Inline*:: This is a function written in {sqlpp} which can be used as part of another {sqlpp} statement (such as xref:n1ql:n1ql-language-reference/selectintro.adoc[`SELECT`] and xref:n1ql:n1ql-language-reference/execfunction.adoc[`EXECUTE FUNCTION`])
```

Comment:
Actually tbh I would move this section down to the discussion of the **Function Type** drop-down list. It would be more useful there.